### DISCUSSION TITLE
How to set a location for an asset?

### DISCUSSION THREAD

**Discussion by designatedsuccessor** [snipe/snipe-it#14684] [Q&A]
This is probably a dumb question, but how do you assign a location to an asset?

When I create a new asset, there is a field called "Default Location" with this description: "This is the location of the asset when it is not checked out"

We are really not even doing "check outs" or whatever.  We are assigning assets to users so they know the asset is under their responsibility.

I just want to assign an asset to a location like "Building F, Room 58" so that we know where it's located. But I still want a specific user to know that it's under their care and responsibility.

**Comment by snipe** [comment]
Hi @loosus - assigning an asset to a user is the same as checking it out to that user. If the user has a location, the assets assigned to that user will inherit that location. Default location would be like if you had a warehouse or office that your assets live in (either for first provisioning or when they are returned due to breakage, employee turnover, contract ending, etc) when they are not assigned. Once they are assigned (“checked out”), they inherit the location of who or what they were assigned to (user or location). When they get unassigned (checked back in) they are assumed to be back at the default location (for re-imaging for the next assignment, etc).

Outside of that relationship, we don’t really have a concept of a “responsible party” that *isn’t* the person it’s checked out to. We’re exploring this as an option for the upcoming v7, but we need to nail down exactly how that would work (for example using the location manager as the responsible party when an asset is checked out to a location.)
